src/assets/css/_main.scss

```diff
@@ -254,6 +254,10 @@ table {
 code {
   border-radius: $global-radius;
   overflow-wrap: break-word;
+
+  &.nobackground {
+    background: none;
```

Comment:
This will need to set `color` for the text to ensure it's visible without a background. 

See this screenshot from running this branch. The text is only visible because I've selected it:

<img width="702" alt="Firefox_Extension_Workshop___Get_help_creating___publishing_Firefox_extensions_" src="https://user-images.githubusercontent.com/1514/90404915-f8160700-e09a-11ea-9679-1d9c34e713f0.png">